pandas/core/generic.py

```diff
@@ -5290,6 +5290,9 @@ def get_dtype_counts(self):
         object     1
         dtype: int64
         """
+        warnings.warn("`get_dtype_counts` has been deprecated and will be "
```

Comment:
can you update the docstring and add deprecated

Reply:
Can we recommend `.dtypes.value_counts()` here instead? Or... we're in `generic.py` so that may be too hard?

Reply:
Yeah unfortunately that solution does not work for `Series`, but I could add `for DataFrames use .dtypes.value_counts()`

Reply:
sure, just need something as a replacement (may also want to add in the doc-string itself)